content/blog/pulumi-loves-python/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Pulumi Loves Python"
```

Comment:
Feels a bit vague to me. I guess I wouldn't parse this as a launch blog post and may have skipped it.

Reply:
Maybe: Enhancing Python Support in Pulumi: Poetry, Type Checking, and More!

Reply:
Thanks! The linter complained that this was too long, so I dropped the `in Pulumi` bit.

Reply:
Not saying you need to do that, but you can override with `allow_long_title: true`